### PR TITLE
Feature/cheerio

### PR DESCRIPTION
**Pull request by chungyau97**

### CODE REVIEW

File: integrations/document-loaders/cheerio-web-scraper.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+1.  _(Optional)_ Connect **[Text Splitter](../text-splitters/)**.
+2. Input desired URL to be scrape.
+
+## Crawl and scrape multiple pages
+1. Click **Additional Parameters**.
+2. Select `Web Crawl` or `Scrape XML Sitemap` in **Get Relative Links Method**.
+3. Input `0` in **Get Relative Links Limit** to retrieve all links available in the provided URL.
+
+## Mange Links
```
Comment (0xi4o) [inline integrations/document-loaders/cheerio-web-scraper.md:15]: Typo: "Manage Links"

File: integrations/document-loaders/cheerio-web-scraper.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+1. Click **Manage Links**.
+2.  _(Optional)_ Input dsired URL to be scrape.
```
Comment (0xi4o) [inline integrations/document-loaders/cheerio-web-scraper.md:17]: Typo: "desired"

**Comment by 0xi4o** [comment]
@chungyau97 Can you also mention that the documentation applies for `Puppeteer` and `Playwright` as well? They steps are exactly the same.

**Review by 0xi4o — CHANGES_REQUESTED** [review]
1. Couple of typos
2. Mention that the same steps apply for `Puppeteer` and `Playwright`.

**Comment by chungyau97** [comment]
@0xi4o, which do you think will be the best?
Re-create the same steps in `Puppeteer` and `Playwright`
or
`Cheerio`, `Puppeteer` and `Playwright` reference to another page on how to scrape multiple links.

**Comment by 0xi4o** [comment]
@chungyau97 Referencing `Cheerio` in `Puppeteer` and `Playwright` would be better since we don't have to copy over the exact same steps and have one source of truth.

**Review by 0xi4o — APPROVED** [review]

**Pull request merged**